store/storetest/emoji_store.go

```diff
@@ -74,7 +74,11 @@ func testEmojiGet(t *testing.T, ss store.Store) {
 	}
 
 	for i, emoji := range emojis {
-		emojis[i] = *store.Must(ss.Emoji().Save(&emoji)).(*model.Emoji)
+		data, err := ss.Emoji().Save(&emoji)
+		if err != nil {
```

Comment:
We can replace this with `require.Nil(t, err)`, is more explicit.

Reply:
I've updated the PR to use `require.Nil(t, err). Seems to be a useful testing package, never used it before 👍